packages/create-cli/src/actions/creates.ts

```diff
@@ -1,6 +1,7 @@
 import * as fs from 'fs'
+import * as path from 'path'
 import prompts from 'prompts'
-import normalizeUrl from 'normalize-url'
+import { isValidUrl } from '../utils/directory.js'
```

Comment:
Why `directory.js`? It is a typescript file

Reply:
@umutuzgur , I didn't have the time to check it but it seems there is a problem with TypeScript using ECMAScript Modules. The project is configured as `type: "module"`, and after the transpilation the runtime return the following error running the command if you don't specify the file extension:

![image](https://github.com/checkly/checkly-cli/assets/5315705/f835e339-67e8-43e3-96bb-e6ecabe16cba)

I found all the `/utils` files are imported with the `.js` to work around the issue. I can tackle this later to improve it.